src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -527,7 +527,7 @@ private static ProcessInfo[] GetProcessInfos(PerformanceCounterLib library, int
                                 // at the end.  If instanceName ends in ".", ".e", or ".ex" we remove it.
                                 if (instanceName.Length == 15)
                                 {
-                                    if (instanceName.EndsWith(".", StringComparison.Ordinal))
+                                    if (instanceName.EndsWith('.'))
```

Comment:
Hmm, apparently we don't have this overload for spans.
```suggestion
                                    if (instanceName.EndsWith("."))
```

Reply:
Btw, if you think it's worth it we can unroll EndsWith("literal") too in JIT.
so e.g. `EndsWith(".")` will be properly optimized.

Reply:
why would jit do extra work?